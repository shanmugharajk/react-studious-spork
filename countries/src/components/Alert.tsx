import React from "react";

interface IProps {
  children: React.ReactNode;
}

export const Alert: React.FunctionComponent<IProps> = ({ children }) => (
  <div className="mt-5 p-3 bg-red-50 border border-red-500 rounded-lg">
    <span className="text-red-400">{children}</span>
  </div>
);
