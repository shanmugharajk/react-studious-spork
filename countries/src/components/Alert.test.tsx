import { render } from "@testing-library/react";

import { Alert } from "./Alert";

describe("Alert", () => {
  it("should render Alert", () => {
    const { container } = render(<Alert>Error</Alert>);

    expect(container).toMatchSnapshot();
  });
});
