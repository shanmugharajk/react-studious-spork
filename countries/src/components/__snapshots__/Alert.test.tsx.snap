// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert should render Alert 1`] = `
<div>
  <div
    class="mt-5 p-3 bg-red-50 border border-red-500 rounded-lg"
  >
    <span
      class="text-red-400"
    >
      Error
    </span>
  </div>
</div>
`;
